src/Components/NotesPage/modal/AddNewNoteCard.jsx

```diff
@@ -57,9 +58,9 @@ function AddNewNoteCard({
 				<button
 					title={`${canSubmit ? 'Add to notes' : 'Cannot leave the fields blank'}`}
 					className={`btn ${canSubmit ? 'btn-primary' : 'btn-disabled'} `}
-					onClick={canSubmit ? () => submitNote() : null}
+					onClick={canSubmit ? (isEdit ? () => updateNoteHandler(_id, formData) : () => submitNote()) : null}
```

Comment:
This is slightly confusing. Extract it out.

Reply:
Will extract it after the submmision